### PR TITLE
Adapt to FullHD; do not sort channellist

### PR DESCRIPTION
**Pull request by MegaV0lt**
- Adapt values to FullHD (Affects webif tv preview)
- Do not sort channellist because same channels appear side by side

### CODE REVIEW

**Comment by pbiering** [comment]
can you please resubmit the changes without any ISO-8859-1 / UTF-8 related changes, see also https://github.com/vdr-projects/vdradmin-am/issues/7

File: vdradmind.pl [hunk 1]
```diff
@@ -1164,7 +1165,7 @@ sub get_name_from_uniqid {
 #            return $_->{name} if ($_->{uniq_id} eq $uniq_id);
 #        }
 
-        # Es darf nach Spec nur eine �bereinstimmung geben
+        # Es darf nach Spec nur eine Übereinstimmung geben
```
Comment (pbiering) [inline vdradmind.pl]: Please remove this for this particular PR

File: vdradmind.pl [hunk 2]
```diff
@@ -1178,7 +1179,7 @@ sub get_name_from_vdrid {
         # Kanalliste nach identischer vdr_id durchsuchen
         my @C = grep($_->{vdr_id} == $vdr_id, @{$CHAN{$CHAN_FULL}->{channels}});
 
-        # Es darf nach Spec nur eine �bereinstimmung geben
+        # Es darf nach Spec nur eine Übereinstimmung geben
```
Comment (pbiering) [inline vdradmind.pl]: Please remove this for this particular PR

File: vdradmind.pl [hunk 3]
```diff
@@ -1192,7 +1193,7 @@ sub get_channel_from_vdrid {
...
```
Comment (pbiering) [inline vdradmind.pl]: Please remove this for this particular PR

File: vdradmind.pl [hunk 4]
```diff
@@ -1206,7 +1207,7 @@ sub get_transponder_from_vdrid {
...
```
Comment (pbiering) [inline vdradmind.pl]: Please remove this for this particular PR

File: vdradmind.pl [hunk 5]
```diff
@@ -1220,7 +1221,7 @@ sub get_ca_from_vdrid {
...
```
Comment (pbiering) [inline vdradmind.pl]: Please remove this for this particular PR

File: vdradmind.pl [hunk 6]
```diff
@@ -3031,7 +3032,7 @@ sub epgsearch_Param2Line {
         $directory =~ s/:/\|/g;
     }
 
-    #TODO: $searchtimer_from & $searchtimer_until auf korrektes Format pr�fen
+    #TODO: $searchtimer_from & $searchtimer_until auf korrektes Format prüfen
```
Comment (pbiering) [inline vdradmind.pl]: Please remove this for this particular PR

File: vdradmind.pl [hunk 7]
```diff
@@ -5342,7 +5343,7 @@ sub encode_RecTitle {
         # VFAT on
         for ($i = 0 ; $i < length($title) ; $i++) {
             $c = substr($title, $i, 1);
-            unless ($c =~ /[�������A-Za-z0123456789_!@\$%&()+,.\-;=~ ]/) {
+            unless ($c =~ /[öäüßÖÄÜA-Za-z0123456789_!@\$%&()+,.\-;=~ ]/) {
```
Comment (pbiering) [inline vdradmind.pl]: Please remove this for this particular PR

**Comment by MegaV0lt** [comment]
Pleas check last commit. File is now saved in iso 8859-1

**Pull request merged**